vtex-apps/checkout-cart
# CHK-328: Add log of cart and cartLoaded pixel events

#### What problem is this solving?

Adds the logs for the `cart` and `cartLoaded` events, so RC can capture them and we can properly observe the cart page metrics.

#### How to test it?

[Workspace](https://lucas--checkoutio.myvtex.com/cart/add?sku=289)

Enter the above workspace and observe the network requests. There should be one when the cart page renders for the first time (the `cart` event) and other for when the orderForm completely loads (`carLoaded` event).

#### Screenshots or example usage:

#### Describe alternatives you've considered, if any.

N/A

#### Related to / Depends on

Depends on vtex/pixel-manager#31 and vtex/request-capture-app#14.
